### ISSUE TITLE
Why does tag 0.7.0 exist on this repo?

### ISSUE THREAD

**Issue by talumbau** [open-source-economics/Tax-Calculator#1052]
The current release of `taxcalc` is 0.6.8. @MattHJensen requested a new release because of the new features that have been added since then. In order to do the new release, there are usually only two cases to consider:

 - no new tag exists (so the current release, e.g. "0.6.8", is the most recent tag on the repo). In this case, I simply create a new tag. This new tag will then be associated with the latest commit on master (i.e. HEAD on master)

 - the next tag in the sequence already exists, which may or may not be the same as current HEAD of master. At this point, I just create the release based on that existing tag.

Right now, there is tag "0.6.9" and tag "0.7.0" on this repository. So, it's not clear to me if I should create a release for 0.6.9 or 0.7.0.

So, now that I have explained the two cases, I am wondering the following:
 
 - for which tag should I do the release?

 - was it intentional to create both tag 0.6.9 and 0.7.0?

@MattHJensen do you have guidance here?

**Comment by talumbau** [comment]
cc @martinholmer

**Comment by MattHJensen** [comment]
0.7.0 is the release we hope to deploy to TB. I mentioned this on flowdock but it probably got lost in the flow. 

As an alternative, we could roll the commits from 0.7.0 into 0.6.9 and then delete 0.7.0 and deploy 0.6.9, but I couldn't find a way to do that. 

We have two releases because I didn't communicate effectively w the TC team before releasing 0.6.9 and missed some commits that @martinholmer was hoping to get into TB, so I included those in 0.7.0. 

After 0.7.0, some additional commits have been merged to master, and we specifically _don't_ want those in the release because they have breaking changes that will need to be dealt w in TB and perhaps elsewhere.
